andela/travel_tool_front
# # 161724530 Refactor code and improve test coverage

#### What does this PR do?
-Refactor code for the Travel Readiness Component

#### Description of Task to be completed?
- Write a different middleware and redux actions, reducers to Export Travel Readiness
- Refactor code to reuse already existing Button component
- Remove redundant tests
- Update tests for the Travel Readiness Component

#### How should this be manually tested?
- Clone the repo https://github.com/andela/travel_tool_front.git

- Checkout to the branch `ch-improve-test-coverage-161724530`

- Install dependencies using `yarn install`

- Run tests using `yarn test`. All the tests should be passing
- Start your application using `yarn start`
-Navigate to the dashboard page. You should be able to see the Travel Readiness Card and and on clicking the `export button` you should be able to download the report in CSV format

#### Any background context you want to provide?

You need to be assigned the role of `Travel Admin` or `Super Admin` to view the dashboard page
While reviewing, please ignore `CodeClimate` flag because it's not accurate

#### What are the relevant pivotal tracker stories?
[161724530](https://www.pivotaltracker.com/story/show/161724530)

#### Screenshots (if appropriate)


#### Questions:
N/A
